### PR TITLE
Adjust wait time for vdiff

### PR DESCRIPTION
**Pull request by devpow112**
Trying to sort out some flakiness. So far this makes it work (we'll see how long this holds).

### CODE REVIEW

File: test/perceptual/d2l-rubric-editor.visual-diff.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ describe('d2l-rubric-editor', function() {
 			it('default', async function() {
 				const id = '#default';
 
-				await waitForEntityReady(page, `${id} > d2l-rubric-editor`);
+				await waitForEntityReady(page, `${id} > d2l-rubric-editor`, 6000);
```
Comment (devpow112) [inline test/perceptual/d2l-rubric-editor.visual-diff.js:35]: Wait longer for rubrics editor

File: test/perceptual/d2l-rubric-editor.visual-diff.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ describe('d2l-rubric-editor', function() {
 						it(`${configuration.type} ${kind} ${state}`, async function() {
 							const id = `#${sanitizeName(this.test.title)}`;
 
-							await waitForEntityReady(page, `${id} > d2l-rubric-editor`);
+							await waitForEntityReady(page, `${id} > d2l-rubric-editor`, 6000);
```
Comment (devpow112) [inline test/perceptual/d2l-rubric-editor.visual-diff.js:76]: Wait longer for rubrics editor

File: test/perceptual/helpers/vdiff.js [hunk 3]
```diff
@@ -16,8 +16,8 @@ export const mockPagePostRequests = async(page) => {
 	});
 };
 
-export const waitForEntityReady = async(page, selector) => {
-	await page.evaluate(selector => {
+export const waitForEntityReady = async(page, selector, timeoutMs = 1000) => {
```
Comment (devpow112) [inline test/perceptual/helpers/vdiff.js:19]: Allow passing a wait time, lower default wait time to less than half what it was before.

**Comment by devpow112** [comment]
VDiffs for editor now show `Saved!` instead of `Saving...`. The wait time before seems to be too flaky in terms of it showing `Saved!` vs `Saving...` vs nothing.

**Review by mpharoah-d2l — APPROVED** [review]

**Pull request merged**